src/components/Cards/CardSection.js

```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components';
+import themeGet from 'extensions/themeGet';
+
+const CardSection = styled.div`
+  padding: ${themeGet('spacing.medium')};
+  ${CardSection}:not(:last-of-type) {
+    border-bottom-color: ${themeGet('colors.border.medium')};
+    border-bottom-width: ${themeGet('thicknesses.normal')};
+    border-bottom-style: solid;
+  }
+
+  ${CardSection} > * {
+    margin: 0;
```

Comment:
I'm a little wary of this rule for children, could you describe your reasoning behind it?

Reply:
This was caught in visual inspection; when a paragraph tag is used in the CardSection, the margin on the top and bottom throws off the spacing, as the left and right padding are smaller than the combined top and bottom padding and margin.

Other options are to remove the p tag from usage, or simply let the developer handle the margin.  Ideally, though, the padding is the only spacing around the content, regardless of whatever spacing is specified within the content itself.

Reply:
Seems like we should have a `P` component that removes that browser-default margin if we want to use `<p>`. But I'm ok with this for now. We'll see where things go.